### ISSUE TITLE
[Question] Getting a error for base16-qutebrowser

### ISSUE THREAD

**Issue by BlueInGreen68** [tinted-theming/tinty#53]
**What version of `tinty` are you using?**

```bash
tinty 0.18.0
```

Hi! Thanks for the cool project. I'm having a little trouble applying `tinty` for `qutebrowser`.

For example, I'm trying to put a `base16-3024` circuit and it's in the [base16-qutebrowser](https://github.com/tinted-theming/base16-qutebrowser/tree/main/themes/default) repository, in the `default` folder. When trying to change the scheme `tinty` gives the following error:

```txt
Theme does not exists for base16-qutebrowser in /home/blueingreen68/.local/share/tinted-theming/tinty/repos/base16-qutebrowser/themes. Try running `tinty update` or submit an issue on https://github.com/tinted-theming/tinty
```
Sway in turn changes the subject successfully.

My `~/.config/tinted-theming/tinty/config.toml` file:

```toml
# Global settings
shell = "sh -c '{}'"
default-scheme = "base16-mocha"

# Item configurations
[[items]]
path = "https://github.com/tinted-theming/base16-qutebrowser"
name = "base16-qutebrowser"
themes-dir = "themes"
hook = "cp -f %f ~/.config/qutebrowser/config.d/colorscheme.py"

[[items]]
path = "https://github.com/rkubosz/base16-sway"
name = "base16-sway"
themes-dir = "themes"
hook = "cp -f %f ~/.config/sway/config.d/colorscheme && swaymsg reload"
```

**Comment by JamyGolden** [comment]
Hello :smile: I've checked locally and managed to get it working. The theme path in base16-qutebrowser is slightly different since the themes are in a directory within the `themes/` directory. On top of that they have a different "file extension". Tinty expects `<scheme-system>-<scheme-name>.<extension>`, but qutebrowser has `<scheme-system>-<scheme-name>.config.<extension>` so we need to tell tinty to use `.config.py` as the extension. I've tested the config below and it should work:

```
[[items]]
path = "https://github.com/tinted-theming/base16-qutebrowser"
name = "base16-qutebrowser"
themes-dir = "themes/default"
hook = "echo '%f' && cp -f %f ~/.config/qutebrowser/config.d/colorscheme.py"
theme-file-extension = ".config.py"
```
